### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-5-4

### CODE REVIEW

File: social-app/src/Pages/Auth/Signup/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+     }
+  }
+
+
+
+  useEffect(() =>{
+    if(user){
+      navigate('/home',{replace: true})
+    }
+  },[user,navigate]);
```
Comment (Shailseen) [inline social-app/src/Pages/Auth/Signup/Signup.jsx:36]: Is dependencies of useEffect depend on navigate ?

**Review by Shailseen — COMMENTED** [review]
**Only some suggestion while raise pr:**
- Add some description in Pr.
- Host your project online to check every modules working fine and don't get crashed.
- Folder structures is good.

**Review by shubhamkumar648 — APPROVED** [review]
- I like the way you used redux 
- code is neat 
- good to merge

File: social-app/src/Pages/Auth/Signup/Signup.jsx (hunk 1 above)
Comment (aniketsingh13) [reply social-app/src/Pages/Auth/Signup/Signup.jsx:36]: yes , i think so

**Pull request merged**